ui/ui-frontend/projects/archive-search/src/app/archive/archive-preview/archive-unit-history-tab/lifecycle-event-node/lifecycle-event-node.component.scss

```
@@ -0,0 +1,188 @@
+    text-decoration: underline;
+  }
+
+  i {
+    font-size: 18px;
+  }
+}
+
+.lifecycle-event-details {
+  color: var(--vitamui-grey-900);
+  font-weight: 400;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)